test/finance/vesting/ERC20VestingWallet.test.js

```diff
@@ -0,0 +1,87 @@
+const { constants, expectEvent, expectRevert, time } = require('@openzeppelin/test-helpers');
+const { expect } = require('chai');
+
+const ERC20VotesMock = artifacts.require('ERC20VotesMock');
+const ERC20VestingWallet = artifacts.require('ERC20VestingWallet');
+
+const min = (...args) => args.slice(1).reduce((x, y) => x.lt(y) ? x : y, args[0]);
+
+contract('ERC20VestingWallet', function (accounts) {
```

Comment:
What do you think about defining a behavior instead of repeating the tests?

Reply:
Bump!

Reply:
I gave it a try yesterday, and it was more difficult then I expected. Schedules a regenerated in beforeEach with every new instance, and I wanted to make the behaviour schedule-agnostic, through a generator function.

Reply:
It sounds like we should be able to make it generic with a function `start x duration x timestamp -> number`.